articles/native-platforms/ios-swift.md

```diff
@@ -177,6 +177,10 @@ Then add Lock Facebook's Pod
 pod 'Lock-Facebook', '~> 2.0'
 ```
 
+Click on the `Pods` project and select `Lock-Facebook` target and set `Allow Non-modular Includes In Framework Modules` to Yes:
```

Comment:
@hzalaz Thoughts? This would be a good change to get committed as it has lots of fixes.